.github/workflows/soak.yml

```diff
@@ -59,8 +63,14 @@ jobs:
       - name: Setup comparison variables
         id: comparison
         run: |
-          export COMPARISON_SHA=${{ github.event.pull_request.head.sha }}
-          export COMPARISON_TAG="${{ github.event.pull_request.head.sha }}-${{ github.event.pull_request.head.sha }}"
+          pushd comparison-vector
+          export COMPARISON_SHA=$(git rev-parse HEAD)
+          # push up merge ref to Github so that it is linkable there
+          git checkout -b pr/${{ github.event.number }}/$COMPARISON_SHA
+          git push origin pr/${{ github.event.number }}/$COMPARISON_SHA
```

Comment:
Will this conflict if we have multiple soak runs for the same PR number?

Reply:
Ah, good catch. Not just for multiple runs in the same PR, but if the workflow is restarted it could have the same SHA. I'll update.